_studio/mfx_lib/encode_hw/hevc/agnostic/g12/hevcehw_g12_rext.cpp

```diff
@@ -147,9 +147,10 @@ void RExt::Query1NoCaps(const FeatureBlocks& /*blocks*/, TPushQ1 Push)
             MFX_CHECK(pCO3 && IsRextFourCC(par.mfx.FrameInfo.FourCC), prev(dpar, par));
 
             mfxU32 invalid = 0;
+            bool   bLP     = IsOn(par.mfx.LowPower);
 
-            invalid += CheckOrZero<mfxU16, 12, 0>(pCO3->TargetBitDepthLuma);
-            invalid += CheckOrZero<mfxU16, 12, 0>(pCO3->TargetBitDepthChroma);
+            invalid += bLP * CheckOrZero<mfxU16, 10, 0>(pCO3->TargetBitDepthLuma) + !bLP * CheckOrZero<mfxU16, 12, 0>(pCO3->TargetBitDepthLuma);
```

Comment:
VME doesn't support 10 bit?

Reply:
will if more efficient if using
if(bLP)
{
}
else
{
}

Reply:
Thanks @cchunbo , I have upload a commit to change coding style here.

Reply:
@MaximIntel VME supports 10b but current check is done for IsRextFourCC() cases